packages/sdk/src/types/Promotions.ts

```diff
@@ -0,0 +1,105 @@
+import { OrdersItem } from './Orders'
+import { DiscountUnit, DiscountAmount, DiscountPercent } from './Vouchers'
+import { SimpleCustomer } from './Customers'
+import { ValidationRulesCreateAssignmentResponse } from './ValidationRules'
+
+export interface PromotionsCreateResponse {
+	id: string
+	name: string
+	campaign_type?: 'PROMOTION'
+	type: 'AUTO_UPDATE' | 'STATIC'
+	description: string
```

Comment:
Are you sure that this is indeed always present in response?

Reply:
`description` property is indeed in the promotions create response